README.md

```diff
@@ -352,11 +375,21 @@ Remember you have to set up **starting target project** in *Visual Studio* befor
 To get **Nabla** to be used by an external application *without adding it as a subdirectory*,but still using a submodule, you should perform following:
 
 ```cmake
-list(APPEND NBL_CMAKE_ARGS "-DIRR_BUILD_DOCS:BOOL=OFF") # enable only if you have doxygen installed and detectable by cmake
-list(APPEND NBL_CMAKE_ARGS "-DIRR_BUILD_EXAMPLES:BOOL=OFF")
-list(APPEND NBL_CMAKE_ARGS "-DIRR_BUILD_TOOLS:BOOL=OFF") # the tools don't work yet (Apr 2020 status, might have changed since then)
-list(APPEND NBL_CMAKE_ARGS "-DIRR_BUILD_MITSUBA_LOADER:BOOL=OFF") # you probably don't want this extension
+set(NBL_SOURCE_DIR "<YOUR_NABLA_SOURCE_DIRECTORY>") # PAY ATTENTION: you have to fill this one with Nabla source directory
+set(NBL_BINARY_DIR "${NBL_SOURCE_DIR}/build")
+set(NBL_INSTALL_DIR "${NBL_BINARY_DIR}/install")
+
+list(APPEND NBL_CMAKE_ARGS "-DNBL_BUILD_DOCS:BOOL=OFF") # enable only if you have doxygen installed and detectable by cmake
+list(APPEND NBL_CMAKE_ARGS "-DNBL_BUILD_EXAMPLES:BOOL=OFF")
+list(APPEND NBL_CMAKE_ARGS "-DNBL_BUILD_TOOLS:BOOL=OFF") # the tools don't work yet (Apr 2020 status, might have changed since then)
+list(APPEND NBL_CMAKE_ARGS "-DNBL_BUILD_MITSUBA_LOADER:BOOL=OFF") # you probably don't want this extension
+list(APPEND NBL_CMAKE_ARGS "-DNBL_COMPILE_WITH_SDL2:BOOL=OFF") # you probably don't want this extension
+list(APPEND NBL_CMAKE_ARGS "-D_NBL_COMPILE_WITH_OPEN_EXR_:BOOL=OFF") # you probably don't want this extension
+list(APPEND NBL_CMAKE_ARGS "-D_NBL_COMPILE_WITH_BAW_LOADER_:BOOL=OFF") # you probably don't want this extension
+list(APPEND NBL_CMAKE_ARGS "-D_NBL_COMPILE_WITH_BAW_WRITER_:BOOL=OFF") # you probably don't want this extension
+list(APPEND NBL_CMAKE_ARGS "-D_NBL_COMPILE_WITH_OBJ_LOADER_:BOOL=OFF") # you probably don't want this extension
```

Comment:
they want it, its nr 1 model exchange format

Reply:
okay, I will remove it for future usage, but I have to mark that OBJ will not compile probably since it used BaW invocations being old API and commented out AFAIK

Reply:
it will be restored soon in the NewAPI branch

Reply:
done